testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -113,25 +91,62 @@ def add_api_key_identity(self, name, all_namespaces: bool = False,
         })
 
     @modify
-    def add_anonymous_identity(self, name):
+    def add_anonymous(self, name):
         """Adds anonymous identity"""
-        identities = self.model.spec.setdefault("identity", [])
-        identities.append({"name": name, "anonymous": {}})
+        self.add_item(name, {"anonymous": {}})
+
+    @modify
+    def remove_all(self):
+        """Removes all identities from AuthConfig"""
+        self.section.clear()
+
+
+class MetadataSection(Section, Metadata):
+    """Section which contains metadata configuration"""
+    @modify
+    def add_http_metadata(self, name, endpoint, method: Literal["GET", "POST"]):
+        self.add_item(name, {
```

Comment:
Is it OK not to have documentation here?

Reply:
I believe it is, we have documentation in the interfaces